Formula/red.rb

```diff
@@ -0,0 +1,16 @@
+class Red < Formula
+  desc "Next-gen programming language, strongly inspired by REBOL"
+  homepage "http://www.red-lang.org"
+  url "http://static.red-lang.org/dl/mac/red-061"
```

Comment:
Sorry but I think that I was not clear enough. What we want to install it is exactly and only the binary cross-compiler (red-061 and very soon the red-062). With this binary you can compile any red source file. From now, red can't be bootstrapped that's why rebol2 (non open source project) is used to create the red executable and its author want to distribute it.
Do you accept the pull request?